src/extensions/extensions.ts

```diff
@@ -29,5 +33,89 @@ export class Extensions {
   constructor(readonly app: App) {
     this.client = new ExtensionsApiClient(app);
   }
-  // TODO: Implement this.
+
+  public runtime(): Runtime {
+    return new Runtime(this.client);
+  }
+}
+
+/**
+ * Runtime provides methods to get and modify an Extension Instances runtime data.
+ */
+class Runtime {
```

Comment:
That was my initial thought too. However, API council specifically requested calling this Runtime, so that it is generic enough to house any other methods that are meant to be used by running extensions code